src/core/queryCache.js

```diff
@@ -49,6 +49,8 @@ export function makeQueryCache({ frozen = false, defaultConfig } = {}) {
     globalListeners.forEach(d => d(queryCache, query))
   }
 
+  queryCache.configRef = configRef;
```

Comment:
We just need access to this. We could hide behind an accessor.

```suggestion
  queryCache.getDefaultConfig = () => configRef.current; // could now clone, freeze, etc. to prevent mutation
```

Reply:
I've switched to `getDefaultConfig()` accessor which should allow some flexibility in the future to freeze/clone/etc.